content/objects-and-math/studio/_index.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "Studio: Objects and Math"
+date: 2023-08-31T13:16:22-05:00
+draft: false
+weight: 3
+originalAuthor: Sally Steuterman # to be set by page creator
+originalAuthorGitHub: gildedgardenia # to be set by page creator
+reviewer: John Woolbright # to be set by the page reviewer
+reviewerGitHub: jwoolbright23 # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+In the exercises, you created objects to store data about the candidates for
+our animal astronaut corps. For this studio, we provide you with a ready-made
+set of candidates.
+
+You must create code to:
+
+1. Select the crew.
+1. Perform critical mission calculations.
+1. Determine the fuel required for launch.
+
+## Select the Crew
+
+To access the code for the first part, open `ObjectsStudio01.js` in `javascript-projects/objects-and-math/studio`.
+
+### Randomly Select ID Numbers
+
+Each candidate was assigned an ID number, which is stored in the candidate's
+data file and in the `idNumbers` array.
+
+1. Write a `selectRandomEntry` function to select a random entry from the
+   `idNumbers` array. Review the
+   [Combining Math Methods]({{< relref "../reading/math-applications/#random-selection-from-an-array" >}}) section if you need a
+   reminder on how to do this.
+1. Call the function three times to select three ID numbers. Store these
+   selections in a new array, making sure to avoid repeated numbers. No animal
+   can be selected more than once!
+
+{{% notice green "Tip" "rocket" %}}
+
+   `arrayName.includes(item)` can be used to check if the array already contains
+   `item`. A `while` loop can keep the selection process going until the
+   desired number of entries have been added to the array.
+
+{{% /notice %}}
+
+### Build a `crew` Array
+
+Design a function that takes two arrays as parameters. These hold the randomly
+selected ID numbers and the candidate objects.
+
+Use one or more loops to check which animals hold the lucky ID numbers. They
+will be going on the space mission! Store these animals in a `crew` array,
+and then return that array.
+
+Use a template literal to print, `'____, ____, and ____ are going to space!'`
+Fill in the blanks with the names of the selected animals.
+
+## Orbit Calculations
+
+Open `ObjectStudio02.js` in `javascript-projects/objects-and-math/studio` to work on the next section.
```

Comment:
There's no reference to where a student can find the files mentioned in the Studio. Ideally, this should be added in a setup or getting started section in the book for all assignments, studios and exercises